eng/pipelines/runtime.yml

```diff
@@ -0,0 +1,123 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+    - master
+    - release/*.*
+
+jobs:
+#
+# Checkout repository
+#
+- template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build CoreCLR
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: checked
+    platforms:
+    - OSX_x64
+    - Linux_x64
+    - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm64
```

Comment:
The `platforms` section is identical between all the jobs here. Is there a reason we don't use a variable for this?

Reply:
Since this is a parameter I believe this can get very messy on using a variable. Plus I believe when adding the full matrix and including libraries and installer full logic plus platform matrix in between CI and PRs, they will be somehow different based on what build we're running.